evergreen/config.yml

```diff
@@ -1670,12 +1559,12 @@ buildvariants:
     cmake_generator: Xcode
     max_jobs: $(sysctl -n hw.logicalcpu)
     xcode_developer_dir: /Applications/Xcode13.1.app/Contents/Developer
-    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=arm64 -DREALM_ENABLE_GEOSPATIAL=OFF
+    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=arm64
   tasks:
   - name: compile_test
   - name: swift-build-and-test
 
-- name: macos-1100-arm64-release
+- name: macos-release
```

Comment:
this is the same problem with renaming for benchmarks: we run them on macos-release and macos-1100-arm64-release.
I think we can simply reuse default 'macos' builder for compile_test_and_package, long-running-tests and swift-build-and-test